### PR TITLE
Feature/offline_steps and full_steps upgrade

### PR DESCRIPTION
**Pull request by sebasbad**
Upgrades flutter projects in the `offline_steps` and `full_steps` directories: 
* Updated Android project gradle configuration to be able to run the app from the Android Studio, using an Android simulator .
* Updated iOS project to add the changes that result from running the app from the Android Studio,  using an iOS simulator.

### CODE REVIEW

File: offline_steps/step_0_flutter_create/ios/Flutter/flutter_assets/AssetManifest.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (goderbauer) [inline offline_steps/step_0_flutter_create/ios/Flutter/flutter_assets/AssetManifest.json]: Everything in Flutter/flutter_assets should not be checked into the repo. That directory should be added to the gitignore file.

File: offline_steps/step_0_flutter_create/android/build.gradle [hunk 2]
```diff
@@ -1,35 +1,29 @@
 buildscript {
```
Comment (goderbauer) [inline offline_steps/step_0_flutter_create/android/build.gradle]: The unfortunate thing with this repository is that you'll have to do all these changes to the other steps in the offline_steps and full_steps directories as well.

File: offline_steps/step_0_flutter_create/ios/Flutter/flutter_assets/AssetManifest.json (hunk 1 above)
Comment (sebasbad) [reply offline_steps/step_0_flutter_create/ios/Flutter/flutter_assets/AssetManifest.json]: Ok, i'll add everything in Flutter/flutter_assets to the gitignore file. Thnx !

File: offline_steps/step_0_flutter_create/android/build.gradle (hunk 2 above)
Comment (sebasbad) [reply offline_steps/step_0_flutter_create/android/build.gradle]: I know, but I'll try anyway :)

**Comment by goderbauer** [comment]
Sorry for dropping the ball on this one. I am going to close this since it is over a year old and we should just upgrade the codelab to the latest current version of Flutter.

**Pull request closed without merging**